include/gps/gps_reader.h

```diff
@@ -16,6 +19,9 @@ class GPSReader
     static constexpr const char *DefaultLinuxDevicePath = "/dev/serial/by-id/usb-u-blox_AG_-_www.u-blox.com_u-blox_GNSS_receiver-if00";
     GPSReader(const char *devicePath = DefaultLinuxDevicePath);
 
+    //! Note: This will probably be UTC
+    const std::tm &getCurrentDateTime() const;
```

Comment:
In theory it can be a local time (but I've no idea why anyone would actually *want* their GPS device to do this). You get timezone info in the ``$GNZDA`` messages that we're now parsing, so the code should handle this properly.

It doesn't really matter because we'd likely always want to convert the returned time to the user's localtime anyway (to avoid bugs like #260).

Reply:
ahh gotcha